modules/base/src/properties/volumeinformationproperty.cpp

```diff
@@ -89,7 +89,12 @@ VolumeInformationProperty::VolumeInformationProperty(std::string_view identifier
         },
         props(*this));
 
-    util::for_each_in_tuple([&](auto& p) { addProperty(p); }, meta(*this));
+    util::for_each_in_tuple(
+        [&](auto& p) {
+            p.setCurrentStateAsDefault();
```

Comment:
So that the property states are not serialized. This affects `dataRange_, valueRange_, valueName_, valueUnit_, axesNames_, axesUnits_`. These will be set when loading the data or having a custom override.

Reply:
but will they not already be default? you don't change them after the initialization?